parmed/amber/_amberparm.py

```diff
@@ -387,17 +387,17 @@ def from_structure(cls, struct, copy=False):
         # present as a way to hack entries into the 1-4 pairlist. See
         # https://github.com/ParmEd/ParmEd/pull/145 for discussion. The solution
         # here is to simply set that periodicity to 1.
-        for dt in inst.dihedral_types:
-            if dt.phi_k == 0 and dt.per == 0:
-                dt.per = 1.0
-            elif dt.per == 0:
-                warn('Periodicity of 0 detected with non-zero force constant. '
-                     'Changing periodicity to 1 and force constant to 0 to '
-                     'ensure 1-4 nonbonded pairs are properly identified. This '
-                     'might cause a shift in the energy, but will leave forces '
-                     'unaffected', AmberWarning)
-                dt.phi_k = 0.0
-                dt.per = 1.0
+#        for dt in inst.dihedral_types:
+#            if dt.phi_k == 0 and dt.per == 0:
+#                dt.per = 1.0
+#            elif dt.per == 0:
+#                warn('Periodicity of 0 detected with non-zero force constant. '
+#                     'Changing periodicity to 1 and force constant to 0 to '
+#                     'ensure 1-4 nonbonded pairs are properly identified. This '
+#                     'might cause a shift in the energy, but will leave forces '
+#                     'unaffected', AmberWarning)
+#                dt.phi_k = 0.0
+#                dt.per = 1.0
```

Comment:
I agree and I think this pull request fixes that. I have added three examples. The examples compare the forces from OpenMM with gmx. I don't think it is possible to compare directly with real amber in parmed. Please correct me if I am wrong. The three tests I added are two simple ones (HOOH based, one dihedral, with different parameters) and one (methanol based three dihedrals and 1-4 interaction). Current intermol gives now for methanol:

Energy group summary
=======================================================================
                type     input(gromacs)    output (amber)      diff (amber)
-----------------------------------------------------------------------
Not comparable energies: are likely not to be the same
-----------------------------------------------------------------------
       coulomb total        14.13787136       14.13731760       -0.00055376
          coulomb-14        14.13787136       14.13731760       -0.00055376
              proper         0.01634456        0.01631760       -0.00002696
-----------------------------------------------------------------------
Comparable energy terms: these should be very close
-----------------------------------------------------------------------
               angle         0.19632179        0.19622960       -0.00009219
                bond         0.09382390        0.09372160       -0.00010230
              bonded         0.30649024        0.30626880       -0.00022144
            dihedral         0.01634456        0.01631760       -0.00002696
           nonbonded        14.13787136       14.13731760       -0.00055376
           potential        14.44436160       14.44358640       -0.00077520

I think the differences are because amber output only contains 4 numbers after the decimal point. I don't think it is possible to tell intermol to give out put in kcal/mol. I think it would look better if that was the case.

If someone tells me what mol I need to add proof that this works, please let me know.